c7n/resources/asg.py

```diff
@@ -713,6 +713,7 @@ def process(self, asgs, event=None):
 
 
 @filters.register('progagated-tags')
+@filters.register('propagated-tags')
```

Comment:
unfortunately this filter has been around for approx 8 months so changing the name now is not an option, I've added it as an alias and registered it to prevent backwards compatibility breakage